src/MICore/LaunchOptions.xsd.types.designer.cs

```diff
@@ -387,7 +388,7 @@ public partial class BaseLaunchOptions {
         
         /// <remarks/>
         [System.Xml.Serialization.XmlAttributeAttribute()]
-        public string VisualizerFile;
+        public List<string> VisualizerFile;
```

Comment:
>public List<string> VisualizerFile; [](http://example.com/codeflow?start=8&length=35)

This is a generated file, and you didn't change the schema, so it shouldn't have changed

Reply:
I don't think we need to support this for XML. This is mostly used for VS scenarios and there is already a project system natvis file detector. We mostly need JSON support.

Reply:
Makes sense to me. We just need to backout the changes to this file.